bundles/org.openhab.binding.hydrawise/src/main/resources/OH-INF/thing/channel-types.xml

```diff
@@ -49,13 +53,51 @@
 			<channel id="temperaturehigh" typeId="temperaturehigh"/>
 			<channel id="temperaturelow" typeId="temperaturelow"/>
 			<channel id="conditions" typeId="conditions"/>
-			<channel id="day" typeId="day"/>
+			<channel id="time" typeId="time"/>
 			<channel id="humidity" typeId="humidity"/>
 			<channel id="wind" typeId="wind"/>
+			<channel id="evapotranspiration" typeId="evapotranspiration"/>
+			<channel id="precipitation" typeId="precipitation"/>
+			<channel id="probabilityofprecipitation" typeId="probabilityofprecipitation"/>
+		</channels>
+	</channel-group-type>
+
+	<channel-group-type id="system">
+		<label>System</label>
+		<description>Controller system data</description>
+		<channels>
+			<channel id="name" typeId="name"/>
+			<channel id="summary" typeId="summary"/>
+			<channel id="online" typeId="online"/>
```

Comment:
Is this channel newly introduced or is this just moved? This seems rather the ThingStatus, than a channel.

Reply:
Its new, and makes sense to adjust the ThingStatus, yes,  Thanks!